### PR TITLE
perf: Update OpenAI Embedding with latest embedding model

### PR DESCRIPTION
**Pull request by dciborow**
## Related Issues/PRs

Provide an example for the user of using the latest embedding model if it is available.

## What changes are proposed in this pull request?

_Briefly describe the changes included in this Pull Request._

## How is this patch tested?

- [ ] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.

## Does this PR change any dependencies?

- [x] No. You can skip this section.
- [ ] Yes. Make sure the dependencies are resolved correctly, and list changes here.

## Does this PR add a new feature? If so, have you added samples on website?

- [x] No. You can skip this section.
- [ ] Yes. Make sure you have added samples following below steps.

1. Find the corresponding markdown file for your new feature in `website/docs/documentation` folder.
   Make sure you choose the correct class `estimators/transformers` and namespace.
2. Follow the pattern in markdown file and add another section for your new API, including pyspark, scala (and .NET potentially) samples.
3. Make sure the `DocTable` points to correct API link.
4. Navigate to website folder, and run `yarn run start` to make sure the website renders correctly.
5. Don't forget to add `<!--pytest-codeblocks:cont-->` before each python code blocks to enable auto-tests for python samples.
6. Make sure the `WebsiteSamplesTests` job pass in the pipeline.

### CODE REVIEW

File: notebooks/features/cognitive_services/CognitiveServices - OpenAI Embedding.ipynb [hunk 1]
```diff
@@ -74,6 +74,10 @@
     "deployment_name_embeddings = \"text-search-ada-doc-001\"\n",
     "deployment_name_embeddings_query = \"text-search-ada-query-001\"\n",
     "\n",
+    "# If available, try using the latest embedding model: https://openai.com/blog/new-and-improved-embedding-model\n",
```
Comment (mhamilton723) [inline notebooks/features/cognitive_services/CognitiveServices - OpenAI Embedding.ipynb:77]: Can we just switch this over to this directly and keep the old one in comments?

Comment (dciborow) [reply notebooks/features/cognitive_services/CognitiveServices - OpenAI Embedding.ipynb]: done.

**Comment by dciborow** [comment]
@mhamilton723 , i deployed the 'text-embedding-ada-002' endpoint. This should be all ready for testing.

**Review by mhamilton723 — APPROVED** [review]

**Pull request merged**